### PR TITLE
Make signed great again

### LINKED ISSUE #206: Move to signed ordinals?
What are pros and cons?

### PR DESCRIPTION
**Pull request by aprokop**
Fix #206.

### CODE REVIEW

**Comment by aprokop** [comment]
Need to fix a *bunch* of warnings.

**Comment by aprokop** [comment]
I fixed warnings immediately caused by unsigned int/int comparisons. The decision whether to have *any* unsigned in the interface is out of scope of this PR.

**Comment by sslattery** [comment]
wow that was easy. good thing we did the typedef.

File: packages/Utils/src/DTK_Types.h [hunk 1]
```diff
@@ -27,9 +27,9 @@
 typedef double Coordinate;
 
 //! Local ordinal typedef.
-typedef unsigned int LocalOrdinal;
+typedef int LocalOrdinal;
```
[the remark this answers is not in the document]
Comment (aprokop) [reply packages/Utils/src/DTK_Types.h]: I was debating that. I don't think it matters, but could switch.

**Comment by aprokop** [comment]
Could somebody approve it, or request changes so that we can move forward to backporting to 3.0 branch and then finishing xsdk?

**Pull request merged**